File: cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/exception/CloudOperationNotSupportedException.java
package com.sequenceiq.cloudbreak.cloud.exception;

/**
 * This {@link RuntimeException} is thrown in case an operation is not supported on a Cloud Platfrom.
 */
public class CloudOperationNotSupportedException extends CloudConnectorException {

    public CloudOperationNotSupportedException(String message) {
        super(message);
    }

    public CloudOperationNotSupportedException(String message, Throwable cause) {
        super(message, cause);
    }

    public CloudOperationNotSupportedException(Throwable cause) {
        super(cause);
    }

}

File: cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/exception/CloudOperationNotSupportedException.kt
package com.sequenceiq.cloudbreak.cloud.exception

/**
 * This [RuntimeException] is thrown in case an operation is not supported on a Cloud Platfrom.
 */
class CloudOperationNotSupportedException : CloudConnectorException {

    constructor(message: String) : super(message) {
    }

    constructor(message: String, cause: Throwable) : super(message, cause) {
    }

    constructor(cause: Throwable) : super(cause) {
    }

}